docroot/WEB-INF/src/com/liferay/course/service/impl/CourseServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.course.service.impl;
+
+import com.liferay.course.service.base.CourseServiceBaseImpl;
+
+/**
+ * The implementation of the course remote service.
+ *
+ * <p>
+ * All custom service methods should be put in this class. Whenever methods are added, rerun ServiceBuilder to copy their definitions into the {@link com.liferay.course.service.CourseService} interface.
+ *
+ * <p>
+ * This is a remote service. Methods of this service are expected to have security checks based on the propagated JAAS credentials because this service can be accessed remotely.
+ * </p>
+ *
+ * @author ces
+ * @see com.liferay.course.service.base.CourseServiceBaseImpl
+ * @see com.liferay.course.service.CourseServiceUtil
+ */
+public class CourseServiceImpl extends CourseServiceBaseImpl {
+	/*
+	 * NOTE FOR DEVELOPERS:
+	 *
+	 * Never reference this interface directly. Always use {@link com.liferay.course.service.CourseServiceUtil} to access the course remote service.
+	 */
+}
```

Comment:
@vuong196 
Do not commit *.class files.
service.xml and *impl files should not be committed along with the built files